default.hbs

```diff
@@ -84,6 +84,22 @@ $(document).ready(function () {
     $('.gh-burger').click(function () {
         $('body').toggleClass('gh-head-open');
     });
+
+    // Filter
+    $('.filter-selection').click(function(event) {
+        let classToFilter = event.target.innerHTML.replace(/\s+/g, '-').toLowerCase();
```

Comment:
During my testing, this didn't work with "Arts, entertainment and recreation", possibly because the tag is converted to `tag-arts-entertainment-and-recreation` on Ghost, but your regex here doesn't deal with the comma.